plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -1535,6 +1545,19 @@ public void testLimitPushdown()
                 .isNotFullyPushedDown(LimitNode.class);
     }
 
+    @Test
+    public void testPredicatePushdown()
+    {
+        assertThat(query("SELECT timestamp_col FROM " + ALL_TYPES_TABLE + " WHERE timestamp_col < TIMESTAMP '1971-01-01 00:00:00.000'"))
+                .isFullyPushedDown();
+    }
+
+    @Override
+    public void testCreateTable()
+    {
+        assertQueryFails("CREATE TABLE test_create_table (col INT)", "This connector does not support creating tables");
+    }
```

Comment:
I think we don't need this test doesn't support  `SUPPORTS_CREATE_TABLE` behavior.

Reply:
removed.

Reply:
We shouldn't override. The base class has the behavior check.
https://github.com/trinodb/trino/blob/02fabc7f3ca515f347c0613e29d44ed14d16bdda/testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java#L128-L134